src/test/java/com/fittracker/fittracker/service/WeightServiceTest.java

```diff
@@ -35,16 +38,19 @@
 @ExtendWith(MockitoExtension.class)
 public class WeightServiceTest {
 
-    private static final LocalDate TEST_DATE = LocalDate.of(2023,10,10);
+    private static final LocalDate TEST_DATE = LocalDate.of(2023, 10, 10);
     private static final Double TEST_VALUE = 100.1;
 
     private static final UUID TEST_UUID = UUID.fromString("948cc727-68e5-455c-ab6d-942e585bde0d");
-    private static final Weight WEIGHT = new Weight(TEST_DATE, TEST_VALUE,TEST_UUID);
-    private static final WeightRequest WEIGHT_REQUEST = new WeightRequest(TEST_DATE,TEST_VALUE);
-    private static final WeightResponse WEIGHT_RESPONSE = new WeightResponse(TEST_DATE,TEST_VALUE);
+    private Weight weight = new Weight(TEST_DATE, TEST_VALUE, TEST_UUID);
```

Comment:
If it's not static final anymore, let's move it a bit lower (maybe below weightService), so it's separated from static final variables. Also, no point in initializing it here, if it's done in `beforeEach`.